src/scripts/components/Chart.jsx

```diff
@@ -19,6 +19,15 @@ const renderChart = (opts) => {
   }
 };
 
+const renderNoData = () => (
+  <div className="chart__without-data">
+      Sorry! There&#39;s nothing to show.
```

Comment:
Can we use `&apos;` here since it's easier to read?

Reply:
Yes, Thanks for suggesting this :+1: